Samples/TrendingMovies/TrendingMovies/ImageProcessing/ColorArt.swift

```diff
@@ -319,7 +319,7 @@ private struct RGBADecimalComponents: Hashable {
 
     // Relative luminance formula: https://en.wikipedia.org/wiki/Relative_luminance
     var luminance: CGFloat {
-        0.212_6 * r + 0.715_2 * g + 0.072_2 * b
+        0.2126 * r + 0.7152 * g + 0.0722 * b
```

Comment:
Sorry about these unrelated code formatting changes. SwiftLint changed and we didn't run it on master with the latest version yet. I'm being pragmatic here to not open up a tiny PR this time.

Reply:
i thought these were already in the 8.0.0 branch?   https://github.com/getsentry/sentry-cocoa/pull/2504#issuecomment-1344011248

Reply:
This is a hotfix for the master branch.